MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.md
+include contrib/ **
```

Comment:
This ensures files from the contrib folder are pulled into the module build. This line casts a wide net -- pulling everything under the contrib folder. 

This may be unnecessary , given the changes to setup.py itself.